src/python/turicreate/toolkits/object_detector/object_detector.py

```diff
@@ -262,7 +265,7 @@ def create(dataset, annotations=None, feature=None, model='darknet-yolo',
         'io_thread_buffer_size': 8,
         # TODO: Have the Python side download the Core ML version of the
```

Comment:
You should remove this TODO comment now that it's done.

Reply:
Thanks for the catch!